CVE-2022-3105.yml

```diff
@@ -0,0 +1,371 @@
+    industry would find interesting.
+nickname: NULL Pointer Dereference
+subsystem:
+  name: RDMA
+  note: this is known because of the headers directly referencing rdma, the function names, and the subsystem references.
+  question: >
+    What subsystems was the mistake in? These are WITHIN linux kernel
+
+
+    Determining the subsystem is a subjective task. This is to help us group
+     similar vulnerabilities, so choose a subsystem that other vulnerabilities would be in. Y
+
+    Some areas to look for pertinent information:
+      - Bug labels
+      - Directory names
+      - How developers refer to an area of the system in comments,
+        commit messages, etc.
+
+    Look at the path of the source code files code that were fixed to get
+
+    directory names. Look at comments in the code. Look at the bug reports how
+
+    the bug report was tagged.
+
+
+    Example linux kernel subsystems are:
+      * drivers
+      * crypto
+      * fs
+      * net
+      * lib
+
+    Name should be:
+      * all lowercase English letters
+      * NOT a specific file
+      * can have digits, and _-@/
+
+    Can be multiple subsystems involved, in which case you can make it an array
+
+    e.g.
+        name: ["subsystemA", "subsystemB"] # ok
+        name: subsystemA # also ok
+discovered:
+  answer: 2022-12-13
+  contest: False
+  question: |
+    How was this vulnerability discovered?
+
+    Go to the bug report and read the conversation to find out how this was
+    originally found. Answer in longform below in "answer", fill in the date in
+    YYYY-MM-DD, and then determine if the vulnerability was found by a Google
+    employee (you can tell from their email address). If it's clear that the
+    vulenrability was discovered by a contest, fill in the name there.
+
+    The automated, contest, and developer flags can be true, false, or nil.
+
+    If there is no evidence as to how this vulnerability was found, then please
+    explain where you looked.
+  automated: False
+  developer: True
+discussion:
+  note: I was not able to find any disagreements
+  question: |
+    Was there any discussion surrounding this?
+
+    A discussion can include debates, disputes, or polite talk about how to
+    resolve uncertainty.
+
+    Example include:
+      * Is this out of our scope?
+      * Is this a security?
+      * How should we fix this?
+
+    Just because you see multiple comments doesn't mean it's a discussion.
+    For example:
+      * "Fix line 10". "Ok" is not what we call a discussion
+      * "Ping" (reminding people)
+
+    Check the bugs reports, pull requests, and mailing lists archives.
+
+    These answers should be boolean.
+      discussed_as_security: true or false
+      any_discussion: true or false
+
+    Put any links to disagreements you found in the notes section, or any other
+    comment you want to make.
+  any_discussion: True
+  discussed_as_security: True
+stacktrace:
```

Comment:
I think you just forgot this one. Even if you didn't find a stacktrace you should still put false down